src/trumbowyg.js

```diff
@@ -1276,7 +1276,7 @@ Object.defineProperty(jQuery.trumbowyg, 'defaultOptions', {
 
             var options = {
                 url: {
-                    label: 'URL',
+                    label: t.lang.linkUrl || 'URL',
```

Comment:
why not just `t.lang.url`?

Reply:
because the 'URL' string is used in other locations in your code and if you want to replace those as well, they might collide. eg: image URL

Reply:
Oh, nice catch :)